classes/PodsAdmin.php

```diff
@@ -1360,9 +1366,38 @@ public function admin_setup_edit_proto( $duplicate, $obj ) {
 
 		wp_localize_script( 'pods-dfv', 'podsAdminConfig', $config );
 
+		$this->admin_js_dfv_config();
+
 		pods_view( PODS_DIR . 'ui/admin/setup-edit-proto.php', compact( array_keys( get_defined_vars() ) ) );
 	}
 
+	/**
+	 * Handle setting up the JS DFV config.
+	 *
+	 * @since 2.8.0
+	 */
+	protected function admin_js_dfv_config() {
+		$config = [
+			'wp_locale'      => $GLOBALS['wp_locale'],
+			'currencies'     => PodsField_Currency::$currencies,
+			'datetime'       => [
```

Comment:
* :warning: Array double arrow not aligned correctly; expected 3 space(s) between "'datetime'" and double arrow, but found 7.
* :no_entry_sign: Short array syntax is not allowed

<sub>hash: 8f7c010801a066d12bffbbd580ccc144</sub>